docs/source/streamflow-public-testnet.md

```diff
@@ -0,0 +1,312 @@
+# Streamflow Public Testnet
+
+Transcoding is the process of taking an input video in one format and
+bitrate, and converting it into many formats and bitrates to make it
+playable on the majority of devices on the planet at any connection
+speed.
+
+In the Livepeer network, users who run nodes on their infrastructure
+perform this very important function, and as a result it's important
+that they have high bandwidth connections, sufficient hardware, and
+reliable DevOps practices. These nodes are delegated towards and
+elected to perform this role, and they are rewarded with the ability
+to earn fees from the network.
+
+## Overview
+
+This documentation will refer to two roles that infrastructure providers can play:
+
+1. Orchestrator
+2. Transcoder
+
+An **orchestrator** is a Livepeer-aware, smart, 24/7 process that is responsible to the end user of the network for transcoding jobs being performed correctly. They stake LPT to secure the work that they perform, and ensure it is done correctly. They can be penalized if they maliciously cheat and mistranscode an end users content. When a user starts a Livepeer node on a CPU based server in `-orchestrator` mode, they are operating that process as an orchestrator.
```

Comment:
"Protocol-aware" instead of "Livepeer-aware"

Reply:
Fixed in e0bd1df